src/components/Header/index.scss

```diff
@@ -8,6 +8,7 @@
   }
 
   &__username {
+    padding: 0;
```

Comment:
In the jira ticket, it should have had no blue border on the right side. Did that get changed?

Reply:
I think the JIRA description is written wrong.

The image in JIRA is the bug. It was previously like that and I made sure that the button is fully wrapped.

Reply:
then APPROVED